content/docs/operating/security.md

```diff
@@ -19,6 +19,11 @@ prometheus-team@googlegroups.com. We will fix the issue and coordinate a
 release date with you, acknowledging your effort and mentioning you by name
 if you want.
 
+Prometheus is maintained by volunteers, not by a company. Therefore, fixing on
+security issues is done on a best-effort basis. We strive to release security
+fixes within 7 days for: prometheus, alertmanager, node_exporter,
```

Comment:
These project names are capitalized in the `Authentication, Authorization, and Encryption` section, consistency would be nice.

Reply:
done